docs/shared/react-standalone-tutorial/react-standalone.md

```diff
@@ -120,65 +120,47 @@ Nx uses the following syntax to run tasks:
 
 ![Syntax for Running Tasks in Nx](/shared/images/run-target-syntax.svg)
 
-All targets, such as `serve`, `build`, `test` or your custom ones, are defined in the `project.json` file.
+## Identify Tasks
```

Comment:
I would make this almost a subsection of the "Serving the App". It is like an evolved side-note of why one can just invoke the tasks, but it is not part of the tutorial flow really, which is why I'd make it a subsection.

Should we call this "Inferred Tasks" though?